component-samples/coding-components/samples/angular/src/app/app.component.css

```diff
@@ -1,3 +1,13 @@
+@import url('https://js.arcgis.com/4.30/@arcgis/core/assets/esri/themes/dark/main.css');
```

Comment:
Shouldn't this be in style.css? Either way, all the stylesheet imports should be in one file, not split across multiple files.

Reply:
Good catch. All the CSS should be in component.css rather than applying it globally.